web/templates/message_edit_notice_tooltip.hbs

```diff
@@ -1,6 +1,7 @@
 <div>
-    <div>{{edited_notice_str}}</div>
+    <span>{{t "View edit history"}}</span>
```

Comment:
Yeah, the changes here moving `<div>` to `<p>` and introducing a `<span>` that's really not one further underscores my comment above.